gensim/models/doc2vec_inner.pyx

```diff
@@ -364,12 +364,13 @@ def train_document_dbow(model, doc_words, doctag_indexes, alpha, work=None,
     c.document_len = i
 
     if c.train_words:
+        # single randint() call avoids a big thread-synchronization slowdown
         if model.shrink_windows:
-            # single randint() call avoids a big thread-synchronization slowdown
             for i, item in enumerate(model.random.randint(0, c.window, c.document_len)):
                 c.reduced_windows[i] = item
         else:
-            c.reduced_windows[:] = int(0)
```

Comment:
Sorry if I had previously suggested a syntax that didn't work or didn't make sense (as the underlying structure is MAX_DOCUMENT_LENGTH long) here!

Reply:
No problem, the suggestion made sense as we could have expected the object to be a proper numpy array :)
Then it's just good to see that unit tests are indeed useful!